cmd/snap/cmd_debug_timings.go

```diff
@@ -152,7 +152,9 @@ type timingsData struct {
 	ChangeID       string   `json:"change-id"`
 	EnsureTimings  []Timing `json:"ensure-timings,omitempty"`
 	StartupTimings []Timing `json:"startup-timings,omitempty"`
-	ChangeTimings  map[string]struct {
+
```

Comment:
do we want this extra newline?  TotalDuration belongs with the previous 2, but anyway all of them have complicated relations, easier not to have any newline?

Reply:
Sure, no problem, it was to avoid re-formatting of the lines above.